include/nlohmann/detail/output/serializer.hpp

```diff
@@ -832,6 +833,8 @@ class serializer
 
         // the actual conversion
         // NOLINTNEXTLINE(cppcoreguidelines-pro-type-vararg,hicpp-vararg)
+        char format[100];
```

Comment:
Where is the magic number `100` coming from?

Reply:
By putting this after the `// NOLINT` comment you've broken its association with the `std::snprintf`